src/app/errors/components/ErrorBoundary.spec.tsx

```diff
@@ -41,4 +54,17 @@ describe('ErrorBoundary', () => {
     );
     expect(Sentry.captureException).toHaveBeenCalled();
   });
+
+  it('captures unhandled rejected promises', () => {
+    const tree = renderer.create(<TestContainer>
```

Comment:
maybe this doesn't matter here but <ErrorBoundary> would not normally be wrapped in this container since TestContainer includes Services and MessageProvider

Reply:
I think the ErrorBoundary needs the MessageProvider, I'm not sure if that's always been the case, but it complains if I remove it

Reply:
Yeah, it's probably fine for the purposes of this test - it just technically should be using your new SimpleMessageProvider since ErrorBoundary falls outside of the normal MessageProvider (which gets its language from whatever book is loaded)